client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -0,0 +1,38 @@
+import { SnackbarList, withNotices } from '@wordpress/components';
+import i18n from 'i18n-calypso';
+import { ReactNode } from 'react';
+import type { Pattern } from '../types';
+import './notices.scss';
+
+type NoticesProps = {
+	noticeList: withNotices.Props[ 'noticeList' ];
+	noticeOperations: withNotices.Props[ 'noticeOperations' ];
+	noticeUI?: ReactNode;
```

Comment:
I haven't found a better way to make the `noticeUI` prop optional. Does it look okay?

Reply:
I had to make it optional to avoid a type error because it is mandatory if I use directly `withNotices.Props`. That's why I created `NoticesProps`.

Reply:
Oh, what you want is just to make it optional. My point is we never used it here and why do we want to keep it since you created `NoticesProps`?

If you really want to make it optional, we can use `Optional` from `utility-types`
```ts
import { Optional } from 'utility-types';

type NoticesProps = Optional< withNotices.Props, 'noticeUI' >;
```

But I do think we can just pick what we want here. Does it make sense?

```ts
Pick< withNotices.Props, 'noticeList' | 'noticeOperations' >;
```